examples/task_proc_codegen.rs

```diff
@@ -0,0 +1,7 @@
+use celery::task;
```

Comment:
I think we can leave this example out, right? Could always check the codgen output from the other example

Reply:
The other example doesn't use the macro

Reply:
I think I updated it recently to use the macro